### ISSUE TITLE
Ticker Class not implemented?

### ISSUE THREAD

**Issue by avipersin** [jakevdp/mpld3#341]
```
loc = ticker.MultipleLocator(15.0)
plt.gcf().gca().xaxis.set_major_locator(loc)
```
This should produce a graph where the x-axis contains all integers that are multiples of 15, as described here: http://matplotlib.org/api/ticker_api.html#matplotlib.ticker.MultipleLocator. It works fine on the matplotlib side as you can see with this image: 
![amaps_zonal](https://cloud.githubusercontent.com/assets/8727319/12560492/0e2c3856-c369-11e5-81ce-32cf83f50d97.png)

However, when I convert to a mpld3 interactive graphic it changes the base to 10 and the step is wrong as you can see here:
![capture](https://cloud.githubusercontent.com/assets/8727319/12560586/96785c62-c369-11e5-9ffc-add15d5eae8f.PNG)

If mpld3 doesn't support this, is there a workaround?

**Comment by aflaxman** [comment]
Thanks for identifying.  I will add this to the list of known issues.  As a workaround, you can use a FixedFormatter, e.g. `plt.xticks(range(-90, 91, 15))`. That will be funky when you zoom out, though.

Making this work correctly would be cool, and might proceed analogously to adding the FixedFormatter, which is captured in this commit: https://github.com/jakevdp/mpld3/commit/07abb8052af719d8c60bfc830936ad80a28d9a32

**Comment by avipersin** [comment]
@aflaxman Great. The problem with the FixedFormatter is that it removes the interactivity. An interactive y-axis with a fixed x-axis doesn't look very good.

**Comment by aflaxman** [comment]
Agree, and /cc @etgalloway who just made the Date ticks do interactivity really well, in case he has an inspiration or suggestions for this.

**Comment by etgalloway** [comment]
@avipersin, as a variation on the workaround suggested by @aflaxman, try something like:

```python
ticklocs = range(-20*15, 20*15+1, 15)
axislim = -90, 90
plt.xticks(ticklocs)
plt.xlim(*axislim)
```

This extends the ticks beyond the initial axis limits.  Ticks that would otherwise appear in the figure margins are filtered out (thanks to PR \#345, recently merged into master).
